framework/src/utils/RankThreeTensor.C

```diff
@@ -353,3 +353,15 @@ RankTwoTensor operator*(const RealVectorValue & p, const RankThreeTensor & b)
 
   return result;
 }
+
+RealVectorValue
+RankThreeTensor::doubleContraction(const RankTwoTensor & b) const
```

Comment:
Yeah, for symmetry reasons this should be a double contraction, too. Plus my version is implemented better. I don't want to delete the `*` operator in this PR, because somebody (@mangerij) might use it.